pallets/nft/src/lib.rs

```diff
@@ -166,29 +101,16 @@ pub mod pallet {
 		/// Chain identifier type
 		type ChainId: Parameter + Member + Debug + Default + FullCodec + Into<u8> + From<u8>;
 
-		/// In order to provide generality, we need some way to associate some action on a source chain
-		/// to some action on a destination chain. This may express tokenX on chain A is equivalent to
-		/// tokenY on chain B, or to simply associate that some action performed on chain A should
-		/// result in some other action occurring on chain B. ResourceId is defined as a 32 byte array
-		/// by ChainSafe.
-		type ResourceId: Member
-			+ Default
-			+ FullCodec
-			+ Into<[u8; 32]>
-			+ From<[u8; 32]>
-			+ MaybeSerializeDeserialize
-			+ TypeInfo;
-
 		/// Resource hash id.
 		///
 		/// This type was initially declared in the bridge pallet but was moved here
 		/// to avoid circular dependencies.
 		#[pallet::constant]
-		type HashId: Get<Self::ResourceId>;
+		type HashId: Get<ResourceId>;
```

Comment:
I know this is out of scope of this PR, but do you think we can rename this type to ResourceHashId instead? currently is very un-descriptive.

Reply:
I agree!